code/index.html

```diff
@@ -1,17 +1,115 @@
 <!DOCTYPE html>
 <html>
-  <head>
-    <meta charset="utf-8" />
-    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>Business Site</title>
-    <!-- dont forget to add a css file and link it here! -->
-  </head>
-  <body>
-    <h1>Business name 🌻</h1>
 
-    <!-- video or image as a header is cool :) -->
+<head>
+  <meta charset="utf-8" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <title>Business Site</title>
+  <link rel="stylesheet" type="text/css" href="style.css">
+</head>
 
-    <!-- Signup form -->
+<body>
+  <header>
+    <div class="hero">
+      <logo>
+        <img src="./assets/logo-note.png" alt="the logo is a note">
+      </logo>
+      <div class="header-text">
+        <h1>Concert House</h1>
+        <h2>Become a member and enjoy special offers!</h2>
+      </div>
 
-  </body>
-</html>
+
+      <!-- Navigation of Mobile Phone in hamburger-style-->
+      <div class="navigation">
+        <div class="hamburger-layer"></div>
+        <div class="hamburger-layer"></div>
+        <div class="hamburger-layer"></div>
+        <nav>
+          <ul>
+            <a class="a-header" href="http://google.com"></a>
```

Comment:
The <a> elements within the navigation don’t have visible text. This might confuse users and affect accessibility.